install.sh

```diff
@@ -1,23 +1,32 @@
 #!/bin/sh
+# wpscan-analyze install script for MacOS and Linux. Only works for 64bits devices for now.
 # This script will download, unzip and install binary file to /usr/local/bin/
-# Script requirement: wget, gzip
+# Script requirement: wget
 # More infos https://github.com/lukaspustina/wpscan-analyze
 
 # Install config
-version=v1.0.4
 install_to="/usr/local/bin"
 binary_file="${install_to}/wpscan-analyze"
 
-# Install script
 # Fast fail if any errors
 set -e
+
 # Move to temp dir
 cd /tmp/
+
 # Quit if not a 64 bits arch
 if [ ! "$(uname -m)" = x86_64 ]; then
-    echo "Unsupported architecture, only 64 bits are supported by this script. Please install Rust environment and build wpscan-analayze from source. Check https://github.com/lukaspustina/wpscan-analyze for more infos."
+    echo "Unsupported architecture, only 64 bits are supported by this script. Please install Rust environment and build wpscan-analayze from source. Visit https://github.com/lukaspustina/wpscan-analyze for more infos."
     exit 1
 fi
+
+# Determine version based on lastest git tag
+git clone https://github.com/lukaspustina/wpscan-analyze --quiet 
```

Comment:
How about `curl --silent "https://api.github.com/repos/lukaspustina/wpscan-analyze/releases/latest" | grep tag_name | sed -E 's/.*"v(.*)",/\1/'`

Reply:
Ok I'll use that !